app/constants/colors.js

```diff
@@ -9,6 +9,7 @@ const colors = {
   PRIMARY_TEXT: '#000',
   GREEN: '#32A852',
   VIOLET: '#4051DB',
+  TRANSPARENT: '#00000000',
```

Comment:
does `transparent` work here?

Reply:
@tstirrat, changed to 'transparent' and resolved conflicts